doc/conf.py

```diff
@@ -240,6 +242,37 @@
 
 numpydoc_show_class_members = False
 
+# Make numpydoc to generate plots for example sections
+numpydoc_use_plots = True
+
+plot_pre_code = """
+import networkx as nx
+import numpy as np
+np.random.seed(7)
```

Comment:
I'm not sure what we should do here. On the one hand, having a global randomness setting like this cuts down on boilerplate in the examples; OTOH it also means that the linked code is insufficient to reproduce the image.

I tend to lean towards being explicit and including any necessary seeding in the examples themselves rather than globally, especially since the added boilerplate isn't visible in the docstring (the source code is only shown if a user clicks on the link).

WDYT?

Reply:
Yes, I think this was necessary when the code was visible in the image but that's no longer the case so it'll be better to have all the necessary code to reproduce the images accessible with the source click. So I will go ahead and commit this.